packages/components/src/server-side-render/README.md

```diff
@@ -32,5 +32,25 @@ Output uses the block's `render_callback` function, set when defining the block.
 
 ## API Endpoint
 
-The API endpoint for getting the output for ServerSideRender is `/wp/v2/block-renderer/:block`. It accepts any params, which are used as `attributes` for the block's `render_callback` method.
-
+The API endpoint for getting the output for ServerSideRender is `/wp/v2/block-renderer/:block`. It will use the block's `render_callback` method.
+
+If you pass `attributes` to ServerSideRender you must define these attributes when registering the block.
+
+```php
+register_block_type(
+	'core/archives',
+	array(
+		'attributes' => array(
+			'showPostCounts' => array(
+				'type' => 'boolean',
+				'default' => false,
```

Comment:
The `=>` should be space-aligned in accordance with the WordPress coding standards.